geometry/geometry_config.py

```diff
@@ -789,11 +789,11 @@
     c.NuTauTarget.WallZDim = c.NuTauTarget.BrZ
 
     if (c.NuTauTT.design==4):
-        c.NuTauTT.n_hor_planes = 1
-        c.NuTauTT.n_vert_planes = 1 #simplified
+        c.NuTauTT.n_hor_planes = 4
+        c.NuTauTT.n_vert_planes = 4 #now same number, walls are squared
         c.NuTauTT.scifimat_width = 13.06 * u.cm # old value 13.045
-        c.NuTauTT.scifimat_hor = c.NuTauTarget.WallXDim #13.045 * 7 + endpieces = length of hor. mats
-        c.NuTauTT.scifimat_vert = c.NuTauTarget.WallYDim #13.045 * 11 + endpieces = length of vert. mats
+        c.NuTauTT.scifimat_hor = 55.14 #13.045 * 4 + endpieces (~2.9cm from previous geom) = length of hor. mats
+        c.NuTauTT.scifimat_vert = 55.14 #13.045 * 4 + endpieces (~2.9cm from previous geom) = length of vert. mats
```

Comment:
I would suggest not parametrising this, so that it automatically adapts to changes of the number of planes.

```suggestion
        c.NuTauTT.scifimat_vert = c.NuTauTT.n_vert_planes * 13.045 * u.cm + 2.96 * u.cm  # endpieces (~2.9cm from previous geom)
```